wolfcrypt/src/hmac.c

```diff
@@ -46,7 +46,7 @@
     {
         if (hmac == NULL || (key == NULL && keySz != 0) ||
            !(type == WC_MD5 || type == WC_SHA || type == WC_SHA256 ||
-                type == WC_SHA384 || type == WC_SHA512 || type == BLAKE2B_ID)) {
+                type == WC_SHA384 || type == WC_SHA512)) {
```

Comment:
BLACK2B_ID is removed but SHA3 not added and below BLAKE2B_ID is still being tested.

Reply:
I put blake2b back in and fixed the SHA-3 includes/etc to follow the patterns for the inclusion of Blake2B and SHA-224. I also fixed the issue with the MCAPI build and building without SHA-512.